ga4gh/cli.py

```diff
@@ -414,14 +415,19 @@ def __init__(self, args):
         self._referenceId = args.id
         self._start = args.start
         self._end = args.end
+        self._outputFormat = args.outputFormat
 
     def run(self):
-        iterator = self._client.listReferenceBases(
+        sequence = self._client.listReferenceBases(
             self._referenceId, self._start, self._end)
-        # TODO add support for FASTA output.
-        for segment in iterator:
-            print(segment, end="")
-        print()
+        if self._outputFormat == "text":
+            print(sequence)
+        else:
+            fastaId = "{}".format(self._referenceId) + \
```

Comment:
We try to avoid splitting lines and string concatenation where we can. How about we try something like
```python
start = start if self._start is not None else ""
end = end if self._end is not None else ""
fastaId = "{}:{}-{}".format(self._referenceId, start, end)
```